Gemfile.lock

```diff
@@ -70,7 +70,7 @@ GEM
     bcrypt (3.1.16)
     bootstrap-will_paginate (1.0.0)
       will_paginate
-    brakeman (5.0.1)
+    brakeman (5.0.4)
```

Comment:
Looks like there are a lot of sub dependency changes in this PR - can't really work out why they there are so many as you only changed one direct dependency (`whenever`). May be worth having a separate PR with these in so they are more easily accessible in the commit history, and so there is less scope for any issues if something goes wrong when we merge this PR :)

Reply:
I think it is because I ran bundle update,  now I  narrowed down by running `bundle update whenever`, now it looks better